assembl/models/langstrings.py

```diff
@@ -740,15 +740,15 @@ def __repr__(self):
         if len(value) > 50:
             value = value[:50]+'...'
         if self.error_code:
-            return '%d: [%s, ERROR %d] "%s"' % (
+            return (u'%d: [%s, ERROR %d] "%s"' % (
```

Comment:
The value is unicode, so the format operation has to be on unicode; but notice I encode back to str on the way out, so the output is a str.